src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random;
+
+    public ColorSupplier(Random random) {
+        this.random = random;
+    }
+
+    public String getRandomColor() {
+        int index = random.nextInt(Colors.values().length);
+        return String.valueOf(Colors.values()[index]);
```

Comment:
You can use method name() to get the name of the enum element

Reply:
Done?